blazor/common/playground/getting-started.md

```diff
@@ -11,6 +11,11 @@ documentation: ug
 
 The Blazor playground allows you to develop and test any Blazor component, including both general components and pre-built Syncfusion<sup style="font-size:70%">&reg;</sup> Blazor components.
 
+To get start quickly with Blazor Playground in Visual Studio, you can check on this video.
```

Comment:
Visual Studio?

Reply:
Hi @kmuthukumarmkm ,

I removed the content.

Regards,
@Backiaraj